pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -19,7 +19,7 @@
 
 { lib, fetchurl, writeScript, ruby, kerberos, libxml2, libxslt, python, stdenv, which
 , libiconv, postgresql, v8, clang, sqlite, zlib, imagemagick
-, pkgconfig , ncurses, xapian, gpgme, util-linux, tzdata, icu, libffi
```

Comment:
I don't want to nitpick to much but I think it would be a good idea to move the pkg-config changes to a separate PR for the packages which did not change cmake to nativeBuildInputs. Also moving pkg-config alone should also be a separate PR to keep this one as small as possible.

Reply:
Right, I'll stop performing `pkgconfig` -> `pkg-config` changes for the remainder of this PR and do it in a different one.